### PR TITLE
Update egg-t-s3-manager

### PR DESCRIPTION
**Pull request by gOOvER**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### Changes to an existing Egg:

1. [x] Have you added an explanation of what your changes do and why you'd like us to include them?
2. [x] Have you tested your Egg changes?

### Changelog:

- update all images
- update to PTDL_v2

### CODE REVIEW

File: voice_servers/ts3_manager/egg-t-s3-manager.json [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
         "logs": "{}",
         "stop": "^C"
     },
     "scripts": {
         "installation": {
             "script": "apt -y update\r\napt -y upgrade\r\napt install -y git unzip jq wget tar curl\r\n\r\nexport HOME=\/mnt\/server\r\ncd $HOME\r\n\r\n## get release info and download links\r\nLATEST_RELEASE=$(curl -L -s -H 'Accept: application\/json' https:\/\/github.com\/${GITHUB_PACKAGE}\/releases\/latest)\r\nLATEST_VERSION=$(echo $LATEST_RELEASE | sed -e 's\/.*\"tag_name\":\"\\([^\"]*\\)\".*\/\\1\/')\r\n\r\nif [ -z \"${VERSION}\" ] || [ \"${VERSION}\" == \"latest\" ]; then\r\n    DOWNLOAD_LINK=\"https:\/\/github.com\/${GITHUB_PACKAGE}\/releases\/download\/$LATEST_VERSION\/$MATCH-$LATEST_VERSION\"\r\nelse \r\n    DOWNLOAD_LINK=\"https:\/\/github.com\/${GITHUB_PACKAGE}\/releases\/download\/v$VERSION\/$MATCH-$VERSION-v$VERSION\"\r\nfi\r\n\r\necho $DOWNLOAD_LINK\r\nwget $DOWNLOAD_LINK\r\n\r\nif [ -z \"${VERSION}\" ] || [ \"${VERSION}\" == \"latest\" ]; then\r\n    mv -f $MATCH-$LATEST_VERSION $HOME\/start_ts3-manager\r\n    chmod +x start_ts3-manager\r\nelse\r\n    mv -f $MATCH-v$VERSION $HOME\/start_ts3-manager\r\n    chmod start_ts3-manager\r\nfi",
```
Comment (Software-Noob) [inline voice_servers/ts3_manager/egg-t-s3-manager.json:25]: The new install image [contains all of these packages](https://github.com/parkervcp/yolks/blob/master/installers/debian/Dockerfile#L35) except wget. You can remove them from the install script.

Wget can be replaced with curl. Doesn't really make sense to use both wget and curl

**Review by Software-Noob — CHANGES_REQUESTED** [review]
Everything else LGTM

File: voice_servers/ts3_manager/egg-t-s3-manager.json (hunk 1 above)
Comment (gOOvER) [reply voice_servers/ts3_manager/egg-t-s3-manager.json]: remove package installation and switched to curl

File: voice_servers/ts3_manager/egg-t-s3-manager.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
...
+            "script": "apt update\r\napt -y upgrade\r\n\r\nexport HOME=\/mnt\/server\r\ncd $HOME\r\n\r\n## get release info and download links\r\nLATEST_RELEASE=$(curl -L -s -H 'Accept: application\/json' https:\/\/github.com\/${GITHUB_PACKAGE}\/releases\/latest)\r\nLATEST_VERSION=$(echo $LATEST_RELEASE | sed -e 's\/.*\"tag_name\":\"\\([^\"]*\\)\".*\/\\1\/')\r\n\r\nif [ -z \"${VERSION}\" ] || [ \"${VERSION}\" == \"latest\" ]; then\r\n    DOWNLOAD_LINK=\"https:\/\/github.com\/${GITHUB_PACKAGE}\/releases\/download\/$LATEST_VERSION\/$MATCH-$LATEST_VERSION\"\r\nelse \r\n    DOWNLOAD_LINK=\"https:\/\/github.com\/${GITHUB_PACKAGE}\/releases\/download\/v$VERSION\/$MATCH-$VERSION-v$VERSION\"\r\nfi\r\n\r\necho $DOWNLOAD_LINK\r\ncurl -L -O $DOWNLOAD_LINK\r\n\r\nif [ -z \"${VERSION}\" ] || [ \"${VERSION}\" == \"latest\" ]; then\r\n    mv -f $MATCH-$LATEST_VERSION $HOME\/start_ts3-manager\r\n    chmod +x start_ts3-manager\r\nelse\r\n    mv -f $MATCH-v$VERSION $HOME\/start_ts3-manager\r\n    chmod start_ts3-manager\r\nfi",
```
Comment (Software-Noob) [inline voice_servers/ts3_manager/egg-t-s3-manager.json:25]: The apt update/upgrade is an unnecessary step that'll only waste time. Install image is rebuilt every Monday. The packages and sources are guaranteed to be up to date.

Comment (gOOvER) [reply voice_servers/ts3_manager/egg-t-s3-manager.json:25]: 99% of all images have this. and 3 packages are 20 sec. so not really an time issue ;)

Comment (gOOvER) [reply voice_servers/ts3_manager/egg-t-s3-manager.json:25]: If egg updates are not desired, then please say so, but please do not demand something that you do not do yourself.

**Comment by gOOvER** [comment]
ok, i see. egg updates sind nicht erwünscht. Dann schliesse ich hier mal

**Pull request closed without merging**